subt/docker/base/Dockerfile

```diff
@@ -70,6 +77,9 @@ RUN apt-get update \
 SHELL ["/bin/bash", "-c"]
 
 # build osrf/subt repository
+# ... TODO one day:
```

Comment:
I am not sure about these commented lines but they helped me to get list of missing dependencies??